### ISSUE TITLE
v-for with (item, index) parsing errors

### ISSUE THREAD

**Issue by gcandal** [vuejs/vue-eslint-parser#98]
First of all, I'm sorry if the issue doesn't have the best format but I couldn't guidelines for opening issues.

## Description

We've encountered several false-positives when running ESLint:

```
error  Parsing error: ...  Unexpected token
...
> 1 | for(let [(item,]index) in items);
    |          ^  vue/no-parsing-error
```

This happens with every instance of `v-for="(item, index) in items"`.

## Versions

- Node: v12.19.0
- eslint-plugin-vue: 7.4.0
- ESLint: 7.16.0

## ESLint config (partial)

```json
{
  "env": {
    "browser": true,
    "es6": true,
    "jasmine": true,
    "es2021": true,
    "node": true
  },
  "globals": {
    "document": "readonly",
    "navigator": "readonly",
    "window": "readonly"
  },
  "parser": ".../node_modules/vue-eslint-parser/index.js",
  "parserOptions": {
    "parser": "@babel/eslint-parser",
    "ecmaVersion": 2020,
    "sourceType": "module",
    "requireConfigFile": false,
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "plugins": [
    "promise",
    "node",
    "import",
    "mocha",
    "vue"
  ],
 ...
}
```

**Comment by joamag** [comment]
I believe this PR fixes it https://github.com/vuejs/vue-eslint-parser/pull/97

**Comment by gcandal** [comment]
@ota-meshi thank you for reacting so quickly, closing this.

**Comment by ota-meshi** [comment]
I fixed the bug and published it now.
The version is 7.4.1. It works fine in my environment.

**Comment by gcandal** [comment]
On mine as well 👍
